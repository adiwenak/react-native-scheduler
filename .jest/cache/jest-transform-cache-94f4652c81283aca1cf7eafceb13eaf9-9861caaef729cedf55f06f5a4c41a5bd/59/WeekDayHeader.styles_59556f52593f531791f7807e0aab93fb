7ed0995d0904bf79a248d6d9bd4e9138
Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;

var _reactNative = require("react-native");

var _default = _reactNative.StyleSheet.create({
  outerViewContainer: {
    flexDirection: "row",
    justifyContent: "space-between"
  },
  textStyle: {
    fontSize: 12,
    flex: 1,
    color: "black",
    fontWeight: "bold",
    textAlign: "center"
  }
});

exports.default = _default;
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi9Vc2Vycy9ha3VybmFkaS9Qcm9qZWN0cy9yZWFjdC1uYXRpdmUtc2NoZWR1bGVyL2xpYnJhcnkvQ2FsZW5kYXJNb250aC9XZWVrRGF5SGVhZGVyL1dlZWtEYXlIZWFkZXIuc3R5bGVzLnRzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiI7Ozs7O0FBQUE7O2VBT2Usd0JBQVcsTUFBWCxDQUF5QjtBQUNwQyxFQUFBLGtCQUFrQixFQUFFO0FBQ2hCLElBQUEsYUFBYSxFQUFFLEtBREM7QUFFaEIsSUFBQSxjQUFjLEVBQUU7QUFGQSxHQURnQjtBQUtwQyxFQUFBLFNBQVMsRUFBRTtBQUNQLElBQUEsUUFBUSxFQUFFLEVBREg7QUFFUCxJQUFBLElBQUksRUFBRSxDQUZDO0FBR1AsSUFBQSxLQUFLLEVBQUUsT0FIQTtBQUlQLElBQUEsVUFBVSxFQUFFLE1BSkw7QUFLUCxJQUFBLFNBQVMsRUFBRTtBQUxKO0FBTHlCLENBQXpCLEMiLCJzb3VyY2VzQ29udGVudCI6WyJpbXBvcnQgeyBTdHlsZVNoZWV0LCBUZXh0U3R5bGUsIFZpZXdTdHlsZSB9IGZyb20gXCJyZWFjdC1uYXRpdmVcIlxuXG5pbnRlcmZhY2UgU3R5bGUge1xuICAgIG91dGVyVmlld0NvbnRhaW5lcjogVmlld1N0eWxlXG4gICAgdGV4dFN0eWxlOiBUZXh0U3R5bGVcbn1cblxuZXhwb3J0IGRlZmF1bHQgU3R5bGVTaGVldC5jcmVhdGU8U3R5bGU+KHtcbiAgICBvdXRlclZpZXdDb250YWluZXI6IHtcbiAgICAgICAgZmxleERpcmVjdGlvbjogXCJyb3dcIixcbiAgICAgICAganVzdGlmeUNvbnRlbnQ6IFwic3BhY2UtYmV0d2VlblwiXG4gICAgfSxcbiAgICB0ZXh0U3R5bGU6IHtcbiAgICAgICAgZm9udFNpemU6IDEyLFxuICAgICAgICBmbGV4OiAxLFxuICAgICAgICBjb2xvcjogXCJibGFja1wiLFxuICAgICAgICBmb250V2VpZ2h0OiBcImJvbGRcIixcbiAgICAgICAgdGV4dEFsaWduOiBcImNlbnRlclwiXG4gICAgfVxufSlcbiJdfQ==