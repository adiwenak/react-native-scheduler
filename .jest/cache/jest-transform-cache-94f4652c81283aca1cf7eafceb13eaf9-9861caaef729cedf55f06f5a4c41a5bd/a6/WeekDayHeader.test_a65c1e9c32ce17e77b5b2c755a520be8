f7946d7b3b92f85ce3b3bd3ba9eab54c
var _interopRequireWildcard = require("@babel/runtime/helpers/interopRequireWildcard");

var React = _interopRequireWildcard(require("react"));

var _reactTestRenderer = require("react-test-renderer");

var _WeekDayHeader = require("./WeekDayHeader");

describe("<WeekDayHeader />", function () {
  it("should render with given props", function () {
    var component = (0, _reactTestRenderer.create)(React.createElement(_WeekDayHeader.WeekDayHeader, null)).toJSON();
    expect(component).toMatchSnapshot();
  });
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi9Vc2Vycy9ha3VybmFkaS9Qcm9qZWN0cy9yZWFjdC1uYXRpdmUtc2NoZWR1bGVyL2xpYnJhcnkvQ2FsZW5kYXJNb250aC9XZWVrRGF5SGVhZGVyL1dlZWtEYXlIZWFkZXIudGVzdC50c3giXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6Ijs7QUFBQTs7QUFDQTs7QUFDQTs7QUFFQSxRQUFRLENBQUMsbUJBQUQsRUFBc0IsWUFBQTtBQUMxQixFQUFBLEVBQUUsQ0FBQyxnQ0FBRCxFQUFtQyxZQUFBO0FBQ2pDLFFBQU0sU0FBUyxHQUFHLCtCQUNkLG9CQUFDLDRCQUFELE9BRGMsRUFDSyxNQURMLEVBQWxCO0FBRUEsSUFBQSxNQUFNLENBQUMsU0FBRCxDQUFOLENBQWtCLGVBQWxCO0FBQ0gsR0FKQyxDQUFGO0FBS0gsQ0FOTyxDQUFSIiwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0ICogYXMgUmVhY3QgZnJvbSBcInJlYWN0XCJcbmltcG9ydCB7IGNyZWF0ZSB9IGZyb20gXCJyZWFjdC10ZXN0LXJlbmRlcmVyXCJcbmltcG9ydCB7IFdlZWtEYXlIZWFkZXIgfSBmcm9tIFwiLi9XZWVrRGF5SGVhZGVyXCJcblxuZGVzY3JpYmUoXCI8V2Vla0RheUhlYWRlciAvPlwiLCAoKSA9PiB7XG4gICAgaXQoXCJzaG91bGQgcmVuZGVyIHdpdGggZ2l2ZW4gcHJvcHNcIiwgKCkgPT4ge1xuICAgICAgICBjb25zdCBjb21wb25lbnQgPSBjcmVhdGUoXG4gICAgICAgICAgICA8V2Vla0RheUhlYWRlciAvPikudG9KU09OKClcbiAgICAgICAgZXhwZWN0KGNvbXBvbmVudCkudG9NYXRjaFNuYXBzaG90KClcbiAgICB9KVxufSlcbiJdfQ==