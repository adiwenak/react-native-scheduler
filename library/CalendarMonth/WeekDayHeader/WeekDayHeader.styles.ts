import { StyleSheet, TextStyle, ViewStyle } from "react-native"

interface Style {
    outerViewContainer: ViewStyle
    textStyle: TextStyle
}

export default StyleSheet.create<Style>({
    outerViewContainer: {
        flexDirection: "row",
        justifyContent: "space-between"
    },
    textStyle: {
        fontSize: 12,
        flex: 1,
        color: "black",
        fontWeight: "bold",
        textAlign: "center"
    }
})
