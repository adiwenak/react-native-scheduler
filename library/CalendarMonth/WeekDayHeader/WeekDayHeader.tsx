import * as React from "react"
import { Text, View } from "react-native"
import styles from "./WeekDayHeader.styles"

export const WeekDayHeader = () => {
    const weekDays: string[] = ["S", "M", "T", "W", "T", "F", "S"]

    const weekDayHandler = (day: string, index: number) => (
        <Text key={index} style={styles.textStyle}>
            {day}
        </Text>
    )

    return (
        <View style={styles.outerViewContainer}>
            {weekDays.map(weekDayHandler)}
        </View>
    )
}
