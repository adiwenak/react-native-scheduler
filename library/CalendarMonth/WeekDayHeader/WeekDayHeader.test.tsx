import * as React from "react"
import { create } from "react-test-renderer"
import { WeekDayHeader } from "./WeekDayHeader"

describe("<WeekDayHeader />", () => {
    it("should render with given props", () => {
        const component = create(
            <WeekDayHeader />).toJSON()
        expect(component).toMatchSnapshot()
    })
})
