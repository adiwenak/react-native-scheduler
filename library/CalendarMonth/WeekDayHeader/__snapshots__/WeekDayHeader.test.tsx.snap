// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WeekDayHeader /> should render with given props 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "space-between",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "flex": 1,
        "fontSize": 12,
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    S
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "flex": 1,
        "fontSize": 12,
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    M
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "flex": 1,
        "fontSize": 12,
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    T
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "flex": 1,
        "fontSize": 12,
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    W
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "flex": 1,
        "fontSize": 12,
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    T
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "flex": 1,
        "fontSize": 12,
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    F
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "flex": 1,
        "fontSize": 12,
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    S
  </Text>
</View>
`;
